client/post-editor/editor-drawer/label.jsx

```diff
@@ -3,16 +3,25 @@
  */
 import React, { PropTypes } from 'react';
 
-export default function EditorDrawerLabel( { children } ) {
+/**
+ * Internal dependencies
+ */
+import InfoPopover from 'components/info-popover';
+
+export default function EditorDrawerLabel( { children, labelText, helpText } ) {
 	return (
 		<label className="editor-drawer__label">
 			<span className="editor-drawer__label-text">
+				{ labelText }
+				{ helpText && <InfoPopover position="top left">{ helpText }</InfoPopover> }
 				{ children }
```

Comment:
Shouldn't `children` be a descendent of `.editor-drawer__label` ?

Otherwise I think it'd apply the labeled text styling to the input it wraps.